WinUIGallery/WinUIGallery.DesktopWap.csproj

```diff
@@ -29,6 +29,7 @@
   <ItemGroup>
     <PackageReference Include="ColorCode.Core" />
     <PackageReference Include="CommunityToolkit.WinUI.UI" Version="7.1.2" />
+    <PackageReference Include="CommunityToolkit.WinUI.UI.Animations" Version="7.1.2" />
```

Comment:
Is this because of the home screen fade out?

Reply:
Yeah, and the onload animation mentioned in the reply above.